### ISSUE TITLE
[Feature request] Add terminal command support

### ISSUE THREAD

**Issue by BachiMjavanadze** [Huuums/vscode-folder-templates#131]
Please add support of terminal commands. I meen not creating files/folders but run any terminal command. somthing like this:

```
"folderTemplates.structures": [
    {
      "name": "My Custom Commands",
      "customVariables": [
        "CustomVar",
        "CustomVar2"
      ],
      "command":  echo hello $CustomVar and $CustomVar2
  }
```

It would also be better if we had a `folderTemplates` extension icon in the taskbar, to run commands from the project root or even if no folder is open in VSCode yet.

P.S. There are several command executors, but I have tested them all. they are either abandoned or buggy and have poor support.

**Comment by Huuums** [comment]
Hi @BachiMjavanadze , 

Thanks for the suggestion. 

Could you please show me a command example that you would run every time you create a folder from a template? I don't see this being common. That sounds like functionality that should be in another extension I think.

**Comment by BachiMjavanadze** [comment]
> That sounds like functionality that should be in another extension I think.

You are right but as I mentioned above: There are several command executors, but  they are either abandoned or buggy and have poor support.

> show me a command example

eg 1:

`cd ${selectedFile} && ng g c ${input:my-component} --inline-style=true --inline-template=true --skip-tests=true`

eg 2:

`cd ${fileDirname}; cd ..\\; dotnet aspnet-codegenerator view ${selectedText} Empty -outDir Views\\${input:EnterViewFolderName}`
